src/WebRTComm/PrivateJainSipCallConnector.js

```diff
@@ -544,11 +544,11 @@ PrivateJainSipCallConnector.prototype.processInvitingSipResponseEvent = function
 				this.webRTCommCall.onPrivateCallConnectorCallInProgressEvent();
 			}
 			console.debug("PrivateJainSipCallConnector:processInvitingSipResponseEvent(): 1XX response ignored");
```

Comment:
@ammendonca since you 're at it, I'd suggest also enclosing this console.debug() in an else, since when statusCode == 180 it is not ignored as the log says, so that we avoid spammy logs ;)